pom.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.strimzi.test.container</groupId>
+    <artifactId>test-container</artifactId>
+    <packaging>pom</packaging>
+    <version>0.1.0-SNAPSHOT</version>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/strimzi/test-container/issues</url>
+    </issueManagement>
+
+    <developers>
+        <developer>
+            <name>Maros Orsak</name>
+            <email>morsak@redhat.com</email>
+            <organization>Red Hat</organization>
+            <organizationUrl>https://www.redhat.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+        <test-containers.version>1.15.2</test-containers.version>
+        <maven.surefire.version>3.0.0-M5</maven.surefire.version>
+        <maven.javadoc.version>3.1.0</maven.javadoc.version>
+        <spotbugs.version>4.0.3</spotbugs.version>
+        <log4j.version>2.14.1</log4j.version>
+        <docker-java.version>3.2.7</docker-java.version>
+        <jupiter.version>5.7.0</jupiter.version>
+        <hamcrest.version>2.2</hamcrest.version>
+        <kafka.version>3.0.0</kafka.version>
+        <maven.source.version>3.0.1</maven.source.version>
+        <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
+        <maven.compiler.version>3.8.1</maven.compiler.version>
+        <maven.dependency.version>3.1.1</maven.dependency.version>
```

Comment:
Might be nice to separate different maven plugins, actual dependencies and test dependencies with some comments.